Duplicati/Library/Backend/AzureBlob/AzureBlobBackend.cs

```diff
@@ -131,15 +131,18 @@ public IList<ICommandLineArgument> SupportedCommands
                     new CommandLineArgument("azure_account_name",
                         CommandLineArgument.ArgumentType.String,
                         Strings.AzureBlobBackend.StorageAccountNameDescriptionShort,
-                        Strings.AzureBlobBackend.StorageAccountNameDescriptionLong),
+                        Strings.AzureBlobBackend.StorageAccountNameDescriptionLong,
+                        null, null, null, "This is deprecated, use azure-account-name insted"),
```

Comment:
Thanks.  Can we change the spelling to `instead`?

Reply:
Right. I Added another commit, thanks for the correction.